crates/ruma-events/src/room/message/relation_serde.rs

```diff
@@ -157,7 +157,7 @@ pub(super) enum RelationSerHelper {
     Replacement(ReplacementJsonRepr),
```

Comment:
Is it not needed to mark this as `untagged` too (since the `Replacement` struct now has a `rel_type` manual tag inserted), or equivalently, to remove all the `untagged` annotations + `serde(tag = "rel_type")` on top of this enum?

Reply:
If you look carefully, we are not serializing `Replacement` but `ReplacementJsonRepr` (because the `Replacement` has `new_content` that goes directly in `content` and the rest that goes into `m.relates_to`).

Reply:
Hmm ok, I am bit curious why we need that many `Replacement` types then, but that's a discussion for another day :grinning: